Adyen/Troubleshooting/02_SynchronousProcessing.md

```diff
@@ -0,0 +1,12 @@
+# Synchronous processing
```

Comment:
Vielleicht ein spezifischerer Titel? Z.B.: "Blocked queue in synchronous processing"

Kann man hier unsere übliche Troubleshooting-Struktur umsetzen? S. z.B. https://github.com/Actindo-AG/Documentation/blob/main/BigCommerce/Troubleshooting/01_UploadOfProductFailed.md

Evtl. Screenshots hinzufügen. Wie sieht es aus, wenn die Queue blockiert ist? Gibt es eine Fehlermeldung? Was passiert sonst, i.e. wie weiß der Kunde, dass die Queue blockiert ist?